src/page/AlertingPage.tsx

```diff
@@ -39,7 +39,7 @@ const Alerting = () => {
           Learn more about alerting for Synthetic Monitoring.
         </a>
       </p>
-      {canRead ? <AlertingPageContent /> : <InsufficientPermissions />}
+      {canReadAlerts ? <AlertingPageContent /> : <InsufficientPermissions />}
```

Comment:
I saw the insufficient permissions when I just had the basic 'Viewer' role (I didn't look into why). Should we provide more detail what permissions are missing as there are two places you would need to update to have the right permissions?

![Screenshot of the Synthetic Monitoring Alerts page showing the insufficient permissions banner.](https://github.com/user-attachments/assets/d5cd5c8b-73cc-43bf-ac1d-65bd431e9d1f)

Reply:
Hmm I'm confused as I'm not sure how you got to that screen. 

Users with only the `Viewer` role assigned, should see this:
<img width="1491" alt="image" src="https://github.com/user-attachments/assets/a20bdc8c-6f2d-47f8-a915-94ab829d96c5">
assigned roles:
![image](https://github.com/user-attachments/assets/cd2db2f5-41f6-4325-9d62-0aac085cedf9)

Users without the `Viewer` role, but with the `Alerts reader` role, see the following:
<img width="1481" alt="image" src="https://github.com/user-attachments/assets/d5fadde0-610d-4fe5-9af2-ec5023f98888">
assigned roles:
![image](https://github.com/user-attachments/assets/83958af6-972e-4548-bc7d-59253541db4a)


The difference lies in that `Viewer` assigns `fixed:alerting:reader` by default and without it it needs to be manually added.